src/helm/benchmark/metrics/machine_translation_metrics.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+from sacrebleu import BLEU
+
+from helm.benchmark.adaptation.request_state import RequestState
+from .metric import Metric
+from .metric_name import MetricName
+from .statistic import Stat
+
+
+class MachineTranslationMetric(Metric):
+    """
+    Compute the BLEU score for Machine Translation scenarios. The implementation is based on sacrebleu.
+    """
+
+    def evaluate_instances(self, request_states: List[RequestState]) -> List[Stat]:
+        """
+        Compute the corpus-level metric based on all reqeust_states.
+        """
+
+        bleu = BLEU()
+
+        # Assume there is one referece for each instance. TODO: Support multiple references after adding more scenarios.
```

Comment:
Should we raise a `ValueError` if there isn't exactly one reference?

Reply:
Sure!